migrations/versions/830_add_supplierframework_prefill_declaration.py

```diff
@@ -0,0 +1,25 @@
+"""Add SupplierFramework.prefill_declaration_from_framework_id column and constraints
+
+Revision ID: 830
+Revises: 820
+Create Date: 2017-02-02 14:18:54.512957
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '830'
+down_revision = '820'
+
+from alembic import op
+import sqlalchemy as sa
+
+def upgrade():
+    op.add_column('supplier_frameworks', sa.Column('prefill_declaration_from_framework_id', sa.Integer(), nullable=True))
+    op.create_foreign_key(op.f('fk_supplier_frameworks_prefill_declaration_from_framework_id_frameworks'), 'supplier_frameworks', 'frameworks', ['prefill_declaration_from_framework_id'], ['id'])
```

Comment:
This constraint name seems to be too long for Postgres - it's been truncated to `"fk_supplier_frameworks_prefill_declaration_from_framework_id_fr"` in my DB.,  although the `downgrade` still runs so maybe it's just an issue with psql `\d` display.